File: mario.py
from cs50 import get_int
while True:
    n = get_int("Height: ")
    if n > 0 and n < 9:
        break

# for row
for i in range(n):
    # for blankspace
    for j in range(n-i-1):
        print(" ", end="")
    # for hashes
    for k in range(i+1):
        print("#", end="")
    # for new line
    for l in range(1):
        print("")

File: mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n;

    do
    {
        n = get_int("Height: ");
    }
 //to make sure the input entered should be integer and more than 0 and less than 8
    while (n > 8 || n < 1);


// to initiate new row

    for (int i = 0; i < n; i++)
    {

// for inserting dots to acheive right alignment of hashes
//  for ( int d = 0; d > i - n + 1; d--)
// {
//  printf(".");
//}
// remove dots and make our pyramid right aligned
        for (int d = 0; d > i - n + 1; d--)
        {
            printf(" ");
        }
// to print hash columns
        for (int h = 0; h < i + 1; h++)
        {
            printf("#");
        }
// to keep space
        for (int j = 0 ; j >= 0; j--)
        {
            printf("\n");

        }

    }




}